codecov.yml

```diff
@@ -1,3 +1,10 @@
 flag_management:
   default_rules:
     carryforward: true
+coverage:
+  status:
+    project:
```

Comment:
I think you want something targeting `patch` as well or ignoring it. `project` is for the project as a whole while `patch` is coverage of the changed lines. The `patch` check fails most often.

I think the `patch` defaults to a target of the `project` coverage. So our project sits around 36.3% right now and the `patch` check wants at least 36.3% coverage of the changed lines.

I think a 1% project level threshold would allow project coverage to drop 1% in a commit which is pretty big actually. We are at 36.3% coverage (so anything down to 35.3% is allowed).

We could just disable the patch check depending on the 

Here's more info on the config
https://docs.codecov.com/docs/commit-status#patch-status

We could also disable the annotations on GitHub if desired
https://docs.codecov.com/docs/github-checks#disabling-github-checks-patch-annotations-via-yaml

There's also an unexpected coverage change check
https://docs.codecov.com/docs/commit-status#changes-status
https://docs.codecov.com/docs/unexpected-coverage-changes

Reply:
Okay yea project-wise I think threshold could be done to 0.05% even and we'd be fine...
For patches, I think it still should be auto (matching the project percentage), as we _should_ try and write more tests for each patch